.github/workflows/docs.yml

```diff
@@ -75,22 +71,22 @@ jobs:
           DJL_DISABLE_PROGRESS_BAR: true
         run: bash test_notebook.sh
 
-      - name: Build docs website
-        run: tools/scripts/build-website.sh "${{ github.event.inputs.version-number || 'master' }}"
-
       - name: Configure Deployment AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
+
+      - name: Build docs website
+        run: tools/scripts/build-website.sh "${{ github.event.inputs.version-number || 'master' }}"
```

Comment:
build-website doesn't need aws credentials, why move to here?

Reply:
I am working on section in build-website that would download the previous versions.json and then prepend the new version info to it. I moved the aws credentials up to download the versions.json file